sources/packages/web/src/services/http/InstitutionLocation.ts

```diff
@@ -115,6 +130,10 @@ export class InstitutionLocationApi extends HttpBaseClient {
         this.addAuthHeader(),
       );
     } catch (error) {
+      if (error.response) {
+        this.handleRequestError(error.response?.data?.message);
```

Comment:
you are already checking `error.response`, then there is no need of `?` after `error.response`

Reply:
This is a copy paste. But I will bear the refactoring of it.

Reply:
Refactored.